sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy

```diff
@@ -87,7 +89,7 @@ class ProjectOverviewView extends VerticalLayout{
         projectGrid.addSelectionListener({
             failedQCSamplesView.setVisible(false)
 
-            if(viewModel.selectedProject && viewModel.selectedProject.samplesQcFailed > 0){
+            if(viewModel.selectedProject && viewModel.selectedProject.samplesQcPassed.failingSamples > 0){
```

Comment:
What do you think about changing from `samplesQcPassed` to `samplesQc`? Reading `passed.getFailing()` seems a bit odd to me.

Reply:
I was already thinking the same. Do you think it would make sense to aggregate the two statuses into one in the use case?

Reply:
yes :) you are aggregating in the view anyhow. This can be merged imho